openy_system/assets/js/atc.dev.js

```diff
@@ -316,10 +316,7 @@
           dom[tagnum].appendChild(atcb_list);
         } else {
           atcb_link = dom[tagnum].querySelector('.atcb-link');
-
-          if(dom[tagnum].querySelector('.atcb-list') == undefined) {
```

Comment:
What is the implication of removing this conditional? Wouldn't this create two instances of `atcb_list` if the class already exists?

Reply:
@froboy  in line 308 , we are already checking if it is undefined or not

Reply:
@froboy  thank you for reviewing , if you click on on add-to-calendar and select google calendar sometimes it is taking previous event details for example change date to future date and click on any event you will see the previous date event detail https://www.nwsymca.org/group-exercise?date=2024-05-16&locations=&categories=&cn= , let me know if you have any questions.

Reply:
yeah you are right about that , sorry for confusion on that but this check is not in the main addtocalendarjs and i have tested possible scenarios from my side , it is not creating any new list and  is there any way we can test this one on sandbox using regression tests if we have any ?

Reply:
Sounds good. Thanks for the explanation. Let's test it out.

Reply:
@vamsiymca please add your test steps to the main PR body, then @podarok can approve.